docker-stack-docs/build.rst

```diff
@@ -313,17 +313,17 @@ There are two types of images you can extend your image from:
 
 Naming conventions for the images:
 
-+----------------+-----------------------+---------------------------------+--------------------------------------+
-| Image          | Python                | Standard image                  | Slim image                           |
-+================+=======================+=================================+======================================+
-| Latest default | 3.9                   | apache/airflow:latest           | apache/airflow:slim-latest           |
-+----------------+-----------------------+---------------------------------+--------------------------------------+
-| Default        | 3.9                   | apache/airflow:X.Y.Z            | apache/airflow:slim-X.Y.Z            |
-+----------------+-----------------------+---------------------------------+--------------------------------------+
-| Latest         | 3.9,3.10,3.11,3.12    | apache/airflow:latest-pythonN.M | apache/airflow:slim-latest-pythonN.M |
-+----------------+-----------------------+---------------------------------+--------------------------------------+
-| Specific       | 3.9,3.10,3.11,3.12    | apache/airflow:X.Y.Z-pythonN.M  | apache/airflow:slim-X.Y.Z-pythonN.M  |
-+----------------+-----------------------+---------------------------------+--------------------------------------+
++----------------+-------------------------+---------------------------------+--------------------------------------+
+| Image          | Python                  | Standard image                  | Slim image                           |
++================+=========================+=================================+======================================+
+| Latest default | 3.13                    | apache/airflow:latest           | apache/airflow:slim-latest           |
```

Comment:
Why do we bump from 3.9 to 3.13 by default?

Reply:
Ah .. I see - the docs are wrong currently:

<img width="1124" alt="Screenshot 2025-06-17 at 06 41 27" src="https://github.com/user-attachments/assets/cd04b7ac-dc29-421b-a6d2-0d903eee7cb8" />

Reply:
https://github.com/apache/airflow/pull/51822